src/qml/LoginPage.qml

```diff
@@ -28,50 +30,106 @@ Kirigami.Page {
 
 	title: "Login"
 
-	GridLayout {
-		columns: 2
+	Column {
+		id: mainLayout
 		width: parent.width
-		Layout.fillWidth: true
+		spacing: 50
 
-		// JID field
-		Kirigami.Label {
-			text: qsTr("Your Jabber-ID:")
-		}
-		Controls.TextField {
-			id: jidField
-			text: kaidan.jid
-			placeholderText: qsTr("user@example.org")
-			Layout.fillWidth: true
+		Kirigami.Heading {
+			text: "Login to Kaidan"
+			anchors.horizontalCenter: parent.horizontalCenter
 		}
 
-		// Password field
-		Kirigami.Label {
-			text: qsTr("Your Password:")
-		}
-		Controls.TextField {
-			id: passField
-			text: kaidan.password
-			placeholderText: qsTr("Password")
-			echoMode: TextInput.Password
+		GridLayout {
+			columns: 2
+			width: parent.width
 			Layout.fillWidth: true
+
+			// JID field
+			Kirigami.Label {
+				id: jidLabel
+				text: qsTr("Your Jabber-ID:")
+			}
+			Controls.TextField {
+				id: jidField
+				text: kaidan.jid
+				placeholderText: qsTr("user@example.org")
+				Layout.fillWidth: true
+				selectByMouse: true
+			}
+
+			// Password field
+			Kirigami.Label {
+				text: qsTr("Your Password:")
+			}
+			Controls.TextField {
+				id: passField
+				text: kaidan.password
+				placeholderText: qsTr("Password")
+				echoMode: TextInput.Password
+				Layout.fillWidth: true
+				selectByMouse: true
+			}
+
+			// Connect button
+			Controls.Button {
+				id: connectButton
+				text: isRetry ? qsTr("Retry") : qsTr("Connect")
+				Layout.columnSpan: 2
+				Layout.alignment: Qt.AlignRight
+				onClicked: {
+					// disable the button
+					connectButton.enabled = false;
+					// indicate that we're connecting now
+					connectButton.text = "<i>" + qsTr("Connecting...") + "</i>";
+
+					// connect to given account data
+					kaidan.jid = jidField.text;
+					kaidan.password = passField.text;
+					kaidan.mainConnect();
+				}
+			}
 		}
+		
+
+		Controls.ToolBar {
+			id: serviceBar
+			height: Kirigami.Units.gridUnit * 3
+			anchors.horizontalCenter: parent.horizontalCenter
+			Material.foreground: "transparent"
+			Material.background: "transparent"
+			Material.accent: "transparent"
+
+			Row {
+				spacing: 20
+
+				Controls.ToolButton {
+					height: serviceBar.height
+					onClicked: {
+						jidField.placeholderText = qsTr("username@pod (This will be your Jabber ID)");
```

Comment:
Be uniformly: Don't call it "user@xyz" once and then "username@xzy"

Don't just call it "@pod", use a fitting form e.g. "@diaspora-pod.org".

And remove this text in the clambs. It's not necessary.

And another thing: you could also rename the button to "Jabber/Diaspora-ID" (or instead of this).